src/ExpandablePicker/ExpandablePicker.js

```diff
@@ -209,7 +209,7 @@ module.exports = kind(
 		{from: 'selected.content', to: 'selectedText'},
 
 		// Accessibility
-		{from: 'selected.accessibilityLabel', to: '$.header._accessibilityText'}
+		{from: 'accessibilityText', to: '$.header._accessibilityText'}
```

Comment:
accessibilityText looks like public API very much. Would you please change this private val with _ prefix or use totally different name?